### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by robbinjohansson**
Updated readme bla

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1 @@
-# Hello World
+# Hell World
```
Comment (kjellnestor) [inline readme.md:1]: This is nice
```suggestion
# Hell World is nice!
```

Comment (kjellnestor) [reply readme.md:1]: ok

**Review by kjellnestor — APPROVED** [review]
Cats are nice

File: readme.md (hunk 1 above)
Comment (robbinjohansson) [reply readme.md:1]: wtf you have problems with ma code?

**Pull request merged**